config.js.docker

```diff
@@ -8,6 +8,29 @@ function getParam(name, _default) {
   return argv[name] || process.env[envname] || _default;
 }
 
+function getIntParam(name, _default) {
+  const param = getParam(name, _default);
+  const parsedParam = parseInt(param);
+  if (Number.isInteger(parsedParam)) {
+    return parsedParam;
+  } else {
+    return _default;
```

Comment:
I'd say throw an exception to prevent running with wrong parameters.

Reply:
If this is an optional parameter we must allow the default as `undefined` or `null`. Should we start with this validation in the first line?

```
if (param === undefined || param === null) return param;
```

Reply:
Good one. Thanks!

Reply:
Yeah, I've done the same validation for the integer and boolean methods. Can you see this commit? https://github.com/HathorNetwork/hathor-wallet-headless/pull/82/commits/212631446fdf39a2d87ca54e2c499cbc9884b8c0